src/components/FindMovie/FindMovie.tsx

```diff
@@ -17,41 +61,66 @@ export const FindMovie: React.FC = () => {
               id="movie-title"
               placeholder="Enter a title to search"
               className="input is-dander"
+              value={query}
+              onChange={(event) => {
```

Comment:
You can separate this function)

Reply:
Create separate methods instead of creating inline event handlers.